### ISSUE TITLE
Is eslint@5 officially supported?

### ISSUE THREAD

**Issue by dkrutsko** [webpack-contrib/eslint-loader#230]
Hey there, I noticed that `eslint@5` was finally released. Does `eslint-loader` support this version or do we need to wait for an update?

**Comment by patrickabkarian** [comment]
it seems to work

**Comment by RinMinase** [comment]
It works but it displays warnings upon installing using npm / yarn.

`eslint-loader@2.0.0" has incorrect peer dependency "eslint@>=1.6.0 <5.0.0"`

Since in the [package.json](https://github.com/webpack-contrib/eslint-loader/blob/master/package.json) of eslint-loader (2.0.0) stated:

```
"peerDependencies": {
    "eslint": ">=1.6.0 <5.0.0",
    "webpack": ">=2.0.0 <5.0.0"
}
```

**Comment by joernroeder** [comment]
same here. seems to work but getting the warnings. please clarify the status. thx

**Comment by RinMinase** [comment]
#231

**Comment by RinMinase** [comment]
PR #231 is about this, but it seems to have failed on Travis-CI checks

**Comment by gknapp** [comment]
`eslint` 5.1.0 is out now, seeing the same as others:

`warning " > eslint-loader@2.0.0" has incorrect peer dependency "eslint@>=1.6.0 <5.0.0".`

**Comment by dkrutsko** [comment]
It looks like this has been fixed with [`2.1.0`](https://github.com/webpack-contrib/eslint-loader/releases/tag/2.1.0). Thanks everyone.
